### DISCUSSION TITLE
2022-09-26 PyAerocom Meeting

### DISCUSSION THREAD

**Discussion by dulte** [metno/pyaerocom#745] [Development meetings]
## Last meeting

#739

## PRs 

-  #744
- #741 

- #737
  - A bit more work needs to be done with the tests

## Other issues

- #736 

- #740 

- Make sure everything is on A/B
- No more work done on #719
- Waiting for data for #738

**Comment by lewisblake** [comment]
Kanskje next time: where to apply filters e.g., #751
